dev/io.openliberty.accesslists.internal/src/com/ibm/ws/tcpchannel/internal/package-info.java

```diff
@@ -12,4 +12,10 @@
  * @version 1.0
  */
 @org.osgi.annotation.versioning.Version("1.0")
-package com.ibm.ws.tcpchannel.internal;
\ No newline at end of file
+
+
+@TraceOptions(traceGroups = { "TCPChannel", "HttpDispatcher", "HttpTransport", }, messageBundle = AccessListsConstants.TCP_MESSAGES)
```

Comment:
I think we should just have the `TCPChannel` group here - since this package will be used for all TCP connections (not just HTTP)

Reply:
OK will change.

Reply:
Now @TraceOptions(traceGroup = "TCPChannel", messageBundle = AccessListsConstants.TCP_MESSAGES)